query-connector/src/app/constants.ts

```diff
@@ -336,17 +356,17 @@ export interface Concept {
 /*
  * The expected type of a ValueSet.
  */
-// export interface ValueSet {
-//   valueset_id: string;
-//   valueset_version: string;
-//   valueset_name: string;
-//   author: string;
-//   system: string;
-//   ersdConceptType?: string;
-//   dibbsConceptType: string;
-//   includeValueSet: boolean;
-//   concepts: Concept[];
-// }
+export interface InternalValueSet {
```

Comment:
Can change this once I sync up with Brandon on [2789](https://app.zenhub.com/workspaces/skylight-dibbs-viper-6480ba23b5a07644e0e46d23/issues/gh/cdcgov/phdi/2789)